PokeAlarm/Manager.py

```diff
@@ -999,6 +1006,15 @@ def add_optional_travel_arguments(self, info):
         if self.__api_req['DRIVE_DIST']:
             info.update(**self.get_driving_data(lat, lng))
 
+    # Add gym details to an info object
+    def add_gym_details(self, info, gym_id):
```

Comment:
I guess the reason I don't think it is worth moving it into a function is because right now the function takes up about 6-9 lines of code (depending on how you count) where as it would only be around 6 lines to do it without it. The additional complexity doesn't seem worth the trade-off.